libamiibo/Data/AmiiboTag.cs

```diff
@@ -35,12 +31,59 @@
 using Org.BouncyCastle.Crypto.Parameters;
 using Org.BouncyCastle.Math;
 using Org.BouncyCastle.Security;
-using LibAmiibo.Data.Settings.AppData.Games;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
 
 namespace LibAmiibo.Data
 {
     public class AmiiboTag
     {
+        /// <summary>
+        /// The data placed at the start of a blank NTAG215.
+        /// </summary>
+        private static readonly byte[] tagHeader = new byte[] { 
+            // Tag serial number
+            0x04, 0x01, 0x02, 0x8F, 0x03, 0x04, 0x05, 0x06, 0x04, 
+
+            // Internal value
+            0x48, 
+
+            // Lock bytes
+            0x0F, 0xE0, 
+            
+            // Capability container
+            0xF1, 0x10, 0xFF, 0xEE,
+
+            // Amiibo magic
+            0xA5, 
+
+            // Write counter
+            0x00, 0x00,
+
+            // Figure version (always 0x00)
+            0x00
+        };
+
+        /// <summary>
+        /// The data placed at hex offset 0x208 of a blank NTAG215.
+        /// </summary>
+        private static readonly byte[] tagFooter = new byte[] { 
```

Comment:
It was after the merge that I remembered that this is exactly `NtagHelpers.CONFIG_BYTES`.